CSharpSeleniumWebdriverSample/CSharpSeleniumWebdriverSample/tests/SampleTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using Selenium.Axe;
+
+namespace CSharpSeleniumWebdriverSample.Test
+{
+    [TestClass]
+    [DeploymentItem("samplePage.html")]
+    [DeploymentItem("chromedriver.exe")]
+    [DeploymentItem("geckodriver.exe")]
+    [TestCategory("Integration")]
+    public class SampleTest
+    {
+        private IWebDriver _webDriver;
+        private WebDriverWait _wait;
+        private const string MainElementSelector = "main";
+        private const int TimeOutInSeconds = 20;
+
+        [TestCleanup]
+        public virtual void TearDown()
+        {
+            _webDriver?.Quit();
+            _webDriver?.Dispose();
+        }
+
+
+        [TestMethod]
+        [DataRow("Chrome")]
+        [DataRow("Firefox")]
+        public void RunScanOnGivenElement(string browser)
+        {
+            string samplePageURL = @"src\samplePage.html";
+            string integrationTestTargetFile = Path.GetFullPath(samplePageURL);
+            string integrationTestTargetUrl = new Uri(integrationTestTargetFile).AbsoluteUri;
+            
+            this.InitDriver(browser);
+            LoadTestPage(integrationTestTargetUrl);
+
+            string elementSelector1 = "ul";
+            int expectedNumberOfViolation1 = 2;
+            AxeResult results1 = RunScanOnGivenElementBySelector(elementSelector1);
+            results1.Violations.Should().HaveCount(expectedNumberOfViolation1);
+
+            string elementSelector2 = "ol";
+            int expectedNumberOfViolation2 = 1;
+            AxeResult results2 = RunScanOnGivenElementBySelector(elementSelector2);
+            results2.Violations.Should().HaveCount(expectedNumberOfViolation2);
```

Comment:
remove the validation on second element, as it doesnt provide any other additional usage that first validation provides